src/app/components/media/MediaExpanded.js

```diff
@@ -141,6 +168,27 @@ class MediaExpandedComponent extends Component {
             authorUserName={authorUsername}
           />
         );
+      // } else if (isPender && timelineEnabled) {
+      //   return (
+      //     <div ref={this.getPlayerRef}>
+      //       <Player
+      //         key={`${media.url}#t=${start},${end}`}
+      //         onDuration={d => setPlayerState({ duration: d })}
+      //         onPause={() => setPlayerState({ playing: false })}
+      //         onPlay={() => setPlayerState({ playing: true })}
+      //         onProgress={p => setPlayerState({ progress: p })}
+      //         onReady={this.props.onPlayerReady}
+      //         onTimeUpdate={t => setPlayerState({ time: t })}
+      //         playing={playing}
+      //         start={start}
+      //         end={end}
+      //         gaps={gaps}
+      //         scrubTo={scrubTo}
+      //         seekTo={seekTo}
+      //         url={media.url}
+      //       />
+      //     </div>
+      //   );
```

Comment:
Please remove these lines if you don't plan to uncomment them.

Reply:
fixed